app/src/main/java/com/fstyle/androidtrainning/MainActivity.java

```diff
@@ -2,12 +2,50 @@
 
 import android.support.v7.app.AppCompatActivity;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
 
-public class MainActivity extends AppCompatActivity {
+public class MainActivity extends AppCompatActivity implements View.OnClickListener{
+    private TextView resultTextView;
+    private EditText so1EditText, so2EditText;
+    private Button resultButton;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        initViews();
+        initEvents();
+    }
+
+    private void initViews() {
+        resultTextView = findViewById(R.id.resultTextView);
+        so1EditText = findViewById(R.id.so1EditText);
+        so2EditText = findViewById(R.id.so2EditText);
+        resultButton = findViewById(R.id.resultButton);
+    }
+
+    private void initEvents() {
+        resultButton.setOnClickListener(this);
+    }
+
+    @Override
+    public void onClick(View view) {
+        switch (view.getId()){
+            case R.id.resultButton:
+                cong();
+                break;
+        }
+    }
+
+    private void cong() {
+        int so1 = Integer.parseInt(so1EditText.getText().toString());
+        int so2 = Integer.parseInt(so2EditText.getText().toString());
+
+
+        resultTextView.setText(so1 + so2 + "");
```

Comment:
WARNING: [SetTextI18n] Do not concatenate text displayed with `setText`. Use resource string with placeholders.
 When calling `TextView#setText`
* Never call `Number#toString()` to format numbers; it will not handle fraction separators and locale-specific digits properly. Consider using `String#format` with proper format specifications (`%d` or `%f`) instead.
* Do not pass a string literal (e.g. "Hello") to display text. Hardcoded text can not be properly translated to other languages. Consider using Android resource strings instead.
* Do not build messages by concatenating text chunks. Such messages can not be properly translated.